README.md

```diff
@@ -0,0 +1,69 @@
+[![Build Status](CI_BADGE_URL goes here)](REPO_URL goes here)
```

Comment:
This can be removed when using Github Actions as the status is added in the project header. This made sense to have when using a third-party service like Semaphore CI.

Reply:
Removed in 29970bd.